string text
bool mode
---
string iat_text
string nlp_text